.editorconfig

```diff
@@ -7,3 +7,4 @@ end_of_line = lf
 charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
+max_line_length = 100
```

Comment:
i wasnt aware that editorconfig supported this property. i'm wondering what the behavior is when it "[forces hard line wrapping](https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties#max_line_length)". i'm used to getting a lint message for violations of this and having the opportunity to figure out what would be sensible for a place to break the line. this sounds like it would likely break wherever it can. would it be better to leave this to prettier to figure out for us or resolve manually?

if we do go forward with this, i usually go with a length of 120. it tends to be a middle ground between folks that want longer lines and those that want to have side panels not result in side scrolling. does 100 have any specific significance with these tools?

Reply:
This file was a new discovery for me!

I added this just as a [place to configure prettier](https://prettier.io/docs/en/options.html#print-width). I think if we go for 120 that will be fine too. I was trying to find a middle ground to get prettier to make similar decisions to what's already in the code.

I think what you'll find in practice is that prettier will make a sensible decision about breaking the line well before 120 characters.